src/main/java/com/google/cloud/tools/app/impl/cloudsdk/CloudSdkAppEngineFlexibleStaging.java

```diff
@@ -73,10 +77,20 @@ public void stageFlexible(StageFlexibleConfiguration config) throws AppEngineExc
       // TODO : earlier ones should warn?
       // Copy the JAR/WAR file to staging.
       if (config.getArtifact() != null && config.getArtifact().exists()) {
-        Files.copy(config.getArtifact().toPath(),
-            config.getStagingDirectory().toPath()
-                .resolve(config.getArtifact().toPath().getFileName()),
-            REPLACE_EXISTING);
+        Path destination = config.getStagingDirectory().toPath()
+            .resolve(config.getArtifact().toPath().getFileName());
+        Files.copy(config.getArtifact().toPath(), destination, REPLACE_EXISTING);
+
+        // Update artifact permissions so docker can read it when deployed
+        if (!System.getProperty("os.name").contains("Windows")) {
```

Comment:
So it appears Path is recommended going forward. I'm not sure how this is supposed to happen in Windows... added #91 for a set of Windows related checks we need to do.